manuscript/2-goals.txt

```diff
@@ -64,21 +64,21 @@ This sums it up pretty well because you want your software development cycle to
 
-Say you deploy your application to a VPS with limited resources and no real options to scale it fast. If you predict there is a possibility your application would get featured on Hacker News, Reddit, Product Hunt or Slashdot and the traffic goes through the roof, now what? Do you have a deployment process that is good enough for quickly moving your application to a new hosting provider? Does your application and deployment process support running on multiple nodes? Just to be clear, I'm not saying you should make your deployment process deal with all the possible scenarios since that would be extremely time inefficient. What I'm saying is that you should keep those scenarios in the back of your head while you plan and implement with a "if X happens, I can change this easily to fit it"-mentality. Your deployment process should be agile too and try to make it so you can change things fast and easy.
+Say you deploy your application to a VPS with limited resources and no real options to scale it quickly. If you predict there is a possibility your application would get featured on Hacker News, Reddit, Product Hunt or Slashdot and the traffic goes through the roof, now what? Do you have a deployment process that is good enough for quickly moving your application to a new hosting provider? Does your application and deployment process support running on multiple nodes? Just to be clear, I'm not saying you should make your deployment process deal with all the possible scenarios since that would be extremely time inefficient. What I'm saying is that you should keep those scenarios in the back of your head while you plan and implement with a "if X happens, I can change this easily to fit it"-mentality. Your deployment process should be agile too and try to make it so you can change things fast and easy.
 
 ## 2.5 Release cycles
 
 You can write how much code you like, but if you're not shipping any of it you're not contributing with any value. You and your team should have a goal of how often you want to release code. Whether it be once every month or four times a day, as long as you have a goal and a plan for it. If you do not have a goal for it you can't reach something. So set a goal if you do not have one and try to get there. Knowing your current and your wanted release cycle is important for planning your deployment process.
 
-I> By "shipping code" I refer to deploying it, from finishing a feature in your local environment to getting it out to the production server(s).
+> By "shipping code" I refer to deploying it, from finishing a feature in your local environment to getting it out to the production server(s).
```

Comment:
"I>" is used by Leanpub for formatting information boxes, could you revert this?